neuralpredictors/layers/readouts/factorized.py

```diff
@@ -50,7 +79,7 @@ def __init__(
         else:
             self.register_parameter("bias", None)
 
-        self.initialize(mean_activity)
+        self.initialize()
```

Comment:
```suggestion
        self.initialize(mean_activity)
```
I think we should not change the default behavior of code if we do not have too, as this would break models we already have trained / result in reproducibility issues.